modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -39,39 +39,31 @@ The following new Premium plugins was released alongside {productname} 7.0.
 
 The new Premium plugin, **Revision History**, allows you to track changes made to content in the {productname} editor and restore previous versions of the content.
 
-For information on the **Revision History** plugin, see xref:revisionhistory.adoc[Revision History].
+For information on the **Revision History Premium Plugin**, see xref:revisionhistory.adoc[Revision History].
```

Comment:
How about this? It is shorter and can omit `Premium` which is odd to me
```The new Premium plugin, **Revision History**, allows you to track changes made to content in the {productname} editor and restore previous versions of the content. For information, see xref:revisionhistory.adoc[Revision History].```

Reply:
Might repeat the same for others?

Reply:
I agree with this suggestion. @kemister85 thoughts?

Reply:
I agree with the change to the body, but the standard for how we document the `For more info....` is `For information on the **<Premium plugin name>** plugin, see xref:<plugincode>.adoc[<Premium plugin name>].`

Reply:
fixed in [4dc339a](https://github.com/tinymce/tinymce-docs/pull/3160/commits/4dc339a452b604f08520d6af39db09474e38fcb8)